security/README.md

```diff
@@ -0,0 +1,59 @@
+# Security
+
+Developers working on [CSE](../CSE.md) projects should adhere to industry-recommended standard practices for secure design and implementation of code. For the purposes of our customers, this means our engineers should understand the [OWASP Top 10 Web Application Security Risks](https://owasp.org/www-project-top-ten/), as well as how to mitigate as many of them as possible, using the resources below.
+
+## Quick References
+
+* [Secure Coding Practices Quick Reference](https://owasp.org/www-pdf-archive/OWASP_SCP_Quick_Reference_Guide_v2.pdf)
+
+* [Web Application Security Quick Reference](https://www.owasp.org/images/0/09/OWASP_Web_Application_Security_Quick_Reference_Guide_0.3.pdf)
+
+* [Security Mindset/Creating a Security Program Quick Start](https://www.owasp.org/images/5/53/OWASP_Quick_Start_Guide.pdf)
+
+## Azure DevOps Security
+
+* [Security Engineering DevSecOps Practices](https://www.microsoft.com/en-us/securityengineering/devsecops)
+
+* [Azure DevOps Data Protection Overview](https://docs.microsoft.com/en-us/azure/devops/organizations/security/data-protection?view=azure-devops)
+
+* [Security and Identity in Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/organizations/security/about-security-identity?view=azure-devops)
+
+* [Security Code Analysis](https://secdevtools.azurewebsites.net/)
+
+## OWASP Cheat Sheets
+
+**Note:** OWASP is considered to be the gold-standard in computer security information. OWASP maintains an extensive series of cheat sheets which cover all of the OWASP Top 10 and more. Below, many of the more relevant cheat sheets have been summarized. To view all of the cheat sheets, check out their [Cheat Sheet Index](https://github.com/OWASP/CheatSheetSeries/blob/master/Index.md).
+
+* [Access Control Basics](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Access_Control_Cheat_Sheet.md)
+
+* [Attack Surface Analysis](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Attack_Surface_Analysis_Cheat_Sheet.md)
+
+* [Content Security Policy (CSP)](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Content_Security_Policy_Cheat_Sheet.md)
+
+* [Cross-Site Request Forgery (CSRF) Prevention](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md)
+
+* [Cross-Site Scripting (XSS) Prevention](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.md)
+
+* [Cryptographic Storage](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Cryptographic_Storage_Cheat_Sheet.md)
+
+* [Deserialization](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Deserialization_Cheat_Sheet.md)
+
+* [Docker Security](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Docker_Security_Cheat_Sheet.md)
```

Comment:
nit: I haven't checked, but there might be something around kubernetes as well? Might be worth adding here.

Reply:
There isn't a dedicated sheet for k8s but there are k8s tips inside the Docker Security sheet. I'll adjust the title accordingly.